collections/concepts/access-controls.md

```diff
@@ -12,16 +12,266 @@ width: 1200
 height: 630
 ---
 
-# What are Access Controls?
+# Access Controls
+
+Basis Theory secures sensitive data as [Tokens](/concepts/what-are-tokens) that are stored within an isolated 
+compliant environment, called a [Tenant](/concepts/access-controls#what-are-tenants).
+Your systems interact with Tokens or Services through the Basis Theory API using an API key that is issued to 
```

Comment:
a new user might be confused as to what are you referring to as Services

Reply:
Adding `(e.g. Proxy, Reactors)`